app/assets/scripts/map.js

```diff
@@ -79,6 +86,38 @@ function displayDetailedResourceView(marker) {
       resizeMapListGrid();
     });
 
+    $('.ui.rating')
+    .rating({
+      initialRating: 0,
+      maxRating: 5,
+      onRate: function(value) {
+        if (value !== 0)
+          $('#submit').removeClass('disabled').addClass('active');
+      }
+    });
+
+    $('#submit').click(function(e) {
+    e.preventDefault();
+    var rating = $('#rating-input').rating('get rating');
+    var review = $('#review').val();
+    var time = new Date();
```

Comment:
to keep it consistent, i would set date in the view function using python's `datetime`